src/planwise/client/projects/views.cljs

```diff
@@ -58,20 +58,29 @@
                                #(dispatch [:projects/cancel-new-project])}
           [new-project-dialog]])])))
 
+(defn loading-wheel []
+  [:svg.circular {:viewBox "25 25 50 50"}
+   [:circle.path {:cx 50
+                  :cy 50
+                  :fill "none"
+                  :r 20
+                  :stroke-miterlimit 10
+                  :stroke-width 2}]])
```

Comment:
Ok, but since it's a constant, it can simply be `def`'ed and referenced directly later.